theme/components/editor.scss

```diff
@@ -20,7 +20,6 @@
 
 	.ck-editor__main {
 		background: ck-color();
-		overflow: hidden;
```

Comment:
I've got no idea why it was there, so most likely it was some mistake. I guess we'll be fine.